### PR TITLE
Improve use_touchstone

### LINKED ISSUE #100: Improve `use_touchstone()`
* Don't write `touchstone` directory to `.Rbuildignore` if already present.
* add a `force = FALSE` argument.

### PR DESCRIPTION
**Pull request by assignUser**
Closes #100

### CODE REVIEW

File: R/use.R [hunk 1]
```diff
@@ -2,49 +2,63 @@
 #'
 #' This function will initialize {touchstone} in your package repository, use
 #' from root directory.
+#'
+#' @param overwrite Overwrites files if they exists.
```
Comment (lorenzwalthert) [inline R/use.R:6]:
```suggestion
#' @param overwrite Overwrites files if they exist.
```

**Comment by assignUser** [comment]
@lorenzwalthert everything should be documented due to precommit right? (Also I don't think this repo has the pr-commands workflow^^)

**Comment by lorenzwalthert** [comment]
The roxygenize hook is skipped due to the dependency on {gert}, because that depends on the system library libgit2r, which is not available (and pre-commit does not support system dependencies). And you are right, the /roxygenize command is not working here.

**Comment by assignUser** [comment]
@lorenzwalthert  ah I see. This should do it :D

**Review by lorenzwalthert — APPROVED** [review]

**Pull request merged**